packages/agent/conf/demos/knb/babylon/presentation_definitions/babylon.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "knb-babylon",
+  "purpose": "We hebben informatie nodig van u",
+  "input_descriptors": [
+    {
+      "id": "01aba20d-6b8d-4ad0-95ae-28ec4267179f",
+      "name": "Rekening openen bij Babylon",
+      "purpose": "We hebben informatie verstrekt door het KvK nodig",
```

Comment:
this does not read very nicely. something like:

We hebben inforrmatie nodig dat is verstrekt door de KvK?

Reply:
ok
We hebben de door de KvK verstrekte informatie nodig.